saga/src/main/resources/db/changelog/eventuate-dll/changelog-0001.sql

```diff
@@ -0,0 +1,52 @@
+--liquibase formatted sql
+--changeset bangnguyen:eventuate-dll-0001
+--comment: init eventuate tables
```

Comment:
In our convention, we don't put comments like this. These information should be tracked by git commit information

Reply:
Here is not the comment for tracking file they are the information will be saved to the databasechangelog table of liquibase, I don't know why prevent put information about script because when I check in the database the databasechangelog lack of many basic information, id, author, comment
![image](https://github.com/nashtech-garage/yas/assets/98079369/1cf7424d-9276-4f35-830b-2eb94c94d5e1)